### DISCUSSION TITLE
[New Feature] Restoring Veinholes

### DISCUSSION THREAD

**Discussion by DarkInsignia** [Phobos-developers/Phobos#983] [Ideas and feedback]
Currently I do not believe that veinholes work properly in Red alert 2/YR. Namely it's growth etc.
I believe veinholes could be used in some settings to provide an interesting change to mix up the gameplay and a good variety of gamemodes.

I've not thought of any additional tags to expand upon the old ones, but seeing the growth etc work in the first place would already be a great start.

**Comment by gangster-ppm** [comment]
Vk's NPatch Autumn Edition does have it partially working iirc
